test-data/unit/check-columns.test

```diff
@@ -133,9 +133,11 @@ def foobar(): pass
 
 [builtins fixtures/module.pyi]
 
-[case testColumnUnexpectedKeywordArg]
+[case testColumnUnexpectedOrMissingKeywordArg]
 def f(): pass
 (f(x=1)) # E:2: Unexpected keyword argument "x" for "f"
```

Comment:
In future we might want to point to `x` here.

Reply:
Yeah, added a TODO. I don't think that we can easily find the location of `x` currently.